mypy.ini

```diff
@@ -10,5 +10,9 @@ enable_error_code = ignore-without-code
 # Support namespace packages per https://github.com/python/mypy/issues/14057
 explicit_package_bases = True
 
-# Disable overload-overlap due to many false-positives
-disable_error_code = overload-overlap
+disable_error_code =
+	# Disabled due to many false-positives
+	overload-overlap,
+	# Enable import-untyped if you'd like to be notified about using untyped dependencies or missing stubs packages
+	# You can leave it disabled if your package isn't publicly typed / `py.typed`
```

Comment:
```suggestion
	# Disable import-untyped to avoid widespread failures. Remove when safe or when
	# specific exclusions supersede this broad setting.
```